services/video_chess_indexer/main/file_handler.py

```diff
@@ -0,0 +1,14 @@
+import os
+import shutil
+
+def remove_old_label_file():
+    shutil.rmtree('./files/results/result', ignore_errors=True)
+    
+def read_label_file(path):
+    labels = []
+    with open(path) as file:
+        for line in file:
+            split_line = line.rstrip().split(" ")
+            labels.append(split_line)
+
+    return labels
```

Comment:
Are you happy enough using a mix of function modules and class modules? Or do you wanna go entirely OOP?